kube-derive/tests/ui/missing_required.stderr

```diff
@@ -0,0 +1,23 @@
+error: Missing field `group`
```

Comment:
Shows errors from all the missing required attributes (not stopping at `group` anymore)